automation/src/test/java/com/microsoft/identity/common/test/automation/tasks/AcquireTokenCloud.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.identity.common.test.automation.tasks;
+
+import com.microsoft.identity.common.test.automation.actors.User;
+import com.microsoft.identity.common.test.automation.interactions.CloseKeyboard;
+import com.microsoft.identity.common.test.automation.ui.Main;
+import com.microsoft.identity.common.test.automation.ui.Request;
+import com.microsoft.identity.common.test.automation.ui.identityproviders.AADV1.SignInPagePassword;
+import com.microsoft.identity.common.test.automation.ui.identityproviders.AADV1.SignInPageUserName;
+import com.microsoft.identity.common.test.automation.utility.TokenRequest;
+
+import net.serenitybdd.screenplay.Actor;
+import net.serenitybdd.screenplay.Task;
+import net.serenitybdd.screenplay.actions.Click;
+import net.serenitybdd.screenplay.actions.Enter;
+import net.serenitybdd.screenplay.waits.WaitUntil;
+import net.thucydides.core.annotations.Steps;
+
+import static net.serenitybdd.screenplay.matchers.WebElementStateMatchers.isVisible;
+
+public class AcquireTokenCloud implements Task{
+
+    @Steps
+    CloseKeyboard closeKeyboard;
+
+    String prompt;
+    String userIdentifier;
+
+    @Override
+    public <T extends Actor> void performAs(T actor) {
+        User user = (User)actor;
+        TokenRequest tokenRequest = user.getTokenRequest();
+        tokenRequest.setPromptBehavior(prompt);
+        tokenRequest.setUserIdentitfier(userIdentifier);
```

Comment:
if useridentifier is provided, we EnterUserNameForSignInDisambiguation(), is not needed below. You can check my PR and replicate the change here

Reply:
Change here : https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1179/files#diff-34fc433707a6c45470c9357295f698d9R45